source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,250 @@
+any of the compatible languages and use code-server as an IDE. For each of the languages you want to use you need two things: an installation of the specific 
+interpreter and an extension in code-server that allows you to connect to it. The extensions can be found in the 'extensions' menu. In what follows, the steps for
+both Python and R are described. 
+
+Python
+++++++
+
+There are multiple Python extensions available, so feel free to try and install the extension that suits you the best. This comes with the warning that only the
+Microsoft Python extension has been tested by our team. To install this extension, go to 'Extensions' and search for 'Pythonn'. Install the one with as developer
+'ms-python'. If you now open a script, you can now use code-server as an IDE and run the lines of code from within the script (the shortkey is shift+enter).
+Code-server will start a Python session with the currently selected Python interpreter. If you did not specify another one, this should default to the loaded Python
+module. This Python extension gives you the possibility to choose other interpreters as well. In the right down corner, you can see <python-version-number> right next
+to 'Python'. If you click that, a window will appear where you can select your Python version. Next to the module version, you should see at least some system Python
+versions (/bin/python). You can also load other modules, or you can also use conda environments here (if you have any conda environments already, you should see
+them here as well).
+
+If you need more information about creating your customized Python environments, have a look `here <https://docs.vscentrum.be/en/latest/software/python_package_management.html>`_. 
+
+**Remarks:**
+
+- Whenever loading a new Python interpreter,you will have to kill your current Python terminal before you will be able to use this new interpreter.
+
+
+R
+++++
+
+While there are also multiple extensions for using R in code-server, these don't all work as they should. The main issue here is that these extensions were not
+built with use on a cluster/server in mind. Also, these extensions are often built with the idea that you are only using one R version in the background, which
+sadly limits the flexibility to switch between different environments, as you can do with Python environments. 
+
+Even with these limitations, there is still a way to work with R in code-server. For this, you need to install the 'R' extension created by Ikuyadeu. Once this
+is installed, you can specify the path in the settings (click the settings wheel of the extension, then click on 'Extension settings'): search for 'R > Rterm: Linux'
+and add the path to your preferred R installation. This can as well be a miniconda R installation. The packages in the environment will then also be available to you.
```

Comment:
```suggestion
and add the path to your preferred R installation. This can also be a miniconda R installation. The packages in the environment will then also be available to you.
```